deny.toml

```diff
@@ -57,6 +59,8 @@ allow = [
     # Required for ring
     "OpenSSL",
     "CC0-1.0",
+    "GPL-3.0",
+    "MPL-2.0",
 ]
```

Comment:
we only want to allow these for the very specific crates that need them, not in general.

I think there is an exception part for licenses: https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html#the-exceptions-field-optional